lib/diameter-util.js

```diff
@@ -61,3 +61,51 @@ exports.messageToColoredString = function(message) {
     messageString += chalk.gray(_.repeat('-', 80));
     return messageString;
 };
+
+var getPathElements = function(path) {
+    var pathElements = [];
+    if (path === undefined || path === null) return pathElements;
+    return path.split('.').map(function(pathElement) {
+        if (pathElement.endsWith(']')) {
+            var parts = pathElement.split('[');
+            var index = parseInt(parts[1].substring(
+                0,
+                parts[1].length - 1
+            ), 10);
+            return {
+                name: parts[0],
+                index: index
+            }
+        } else {
+            return {
+                index: 0,
+                name: pathElement
+            }
+        }
+    });
+};
+
+exports.getAvpValue = function(message, path) {
+    var pathElements = getPathElements(path);
+    if (pathElements.length === 0) return undefined;
+    var firstAvpName = pathElements[0].name;
+    var avps = _.select(message, avp => {
```

Comment:
What's the minimum Node.js version that you'd still like to support? Might want to avoid arrow functions.